web-share/resources/manual-helper.js

```diff
@@ -0,0 +1,71 @@
+// Sets up the page for running manual tests. Automatically creates the
+// instructions (based on the parameters) and the share button.
+function setupManualShareTest(
```

Comment:
I don't see a compelling need to do this, and it would make all the tests more verbose (having to explicitly say the name of each field whenever we call this).